src/lib/util/Games/Slotmachine.ts

```diff
@@ -183,6 +183,14 @@ export class Slotmachine {
 	/** Light colour in the attachment */
 	public static LIGHT_COLOUR = '#FFFFFF';
 
+	/** Success colour in the attachment */
+
+	public static SUCCESS_COLOUR = '#00C853';
+
+	/** Failure colour in the attachment */
+
```

Comment:
```suggestion
```

IIRC TSDoc won't work with the empty line in between but regardless it should be removed